shared/cfg/machines.cfg

```diff
@@ -66,6 +71,11 @@ variants:
         vga = virtio
         vir_domain_undefine_nvram = yes
         pcie_extra_root_port = 2
+        aavmf_path = /usr/share/edk2/aarch64/
```

Comment:
Hi @PaulYuuu Good abstract, but if we can provide one more parameter: `firmware_name = aavmf`, it might help to get proper file path in case level, before vm start.

Reply:
In the beginning, I considered renaming to `edk2_path`, but for compatibility, kept the old usage.
`ovmf` is only for q35 and `aavmf` only for aarch64, they will not interfere with each other. So I think we don't need `firmware_name` so far.